command/acl_token_update.go

```diff
@@ -122,6 +141,10 @@ func (c *ACLTokenUpdateCommand) Run(args []string) int {
 		token.Policies = policies
 	}
 
+	if len(c.roleNames) != 0 || len(c.roleIDs) != 0 {
+		token.Roles = generateACLTokenRoleLinks(c.roleNames, c.roleIDs)
+	}
```

Comment:
One thing this makes me think is that these flags are additive, meaning they only allow adding new roles to the token, but there's no way to remove them.

Perhaps we also need something like `-unset-role-name` and `-unset-role-id`, similar to [`node meta -uset`](https://developer.hashicorp.com/nomad/docs/commands/node/meta/apply#unset). that would remove the given roles from the token?

Reply:
The `-role-id` and `-role-name` flags are like the `-policy` flags: they completely replace the existing polices or roles if set. I've updated the documentation to make that clear.